codemodel/model/common/aspect.ts

```diff
@@ -29,6 +26,9 @@ export interface Aspect extends Metadata {
   /** deprecation information -- ie, when this aspect doesn't apply and why */
   deprecated?: Deprecation;
 
+  /** where did this aspec come from (jsonpath or 'm4:<soemthing>') */
```

Comment:
Aspect (can't do change suggestions on mobile :/)

Reply:
fixed it.